src/views/FarmAuction/components/AuctionLeaderboard/AuctionProgress.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components'
+import { differenceInSeconds } from 'date-fns'
+import { Auction, AuctionStatus } from 'config/constants/types'
+import useRefresh from 'hooks/useRefresh'
+
+const ProgressContainer = styled.div`
+  background-color: ${({ theme }) => theme.colors.input};
+`
+
+const ProgressBar = styled.div<{ progress: number }>`
+  width: ${({ progress }) => `${progress}%`};
+  height: 16px;
+  background-color: ${({ theme }) => theme.colors.secondary};
+`
```

Comment:
Why are we not using the progress bar from the UI Kit?

Reply:
hm, forgot we have it, will replace 👍

Reply:
fixed in d9b9698f103638844c0204aff3913ba74fb5ea43